lib/runtime/action.js

```diff
@@ -185,6 +185,7 @@ export default class Action {
       }
       let serializer = this.container.lookup(`serializer:${ type }`);
       response.body = serializer.serialize(response.body, assign({ action: this }, response.options));
+      response.options.contentType = serializer.contentType;
```

Comment:
I wonder if we should just pass the Response object to the serializer. It feels a bit more future proof (i.e. what if a serializer wants to add some additional headers, like Link headers for pagination or something. If it get's the entire Response object, you could theoretically tack on additional metadata to the Response, and have the serializer render that however it sees fit.